pages/index.tsx

```diff
@@ -1,7 +1,9 @@
-import SoulCounter from "../components/atoms/SoulCounter";
-import useSoulCounter from "../components/atoms/SoulCounter/useSoulCounter";
+import SoulCounter from "@/components/atoms/SoulCounter";
+import useSoulCounter from "@/components/atoms/SoulCounter/useSoulCounter";
 
-export default () => {
+const Index = () => {
```

Comment:
Next js will give a warning in the browser when trying to hot reload unless you name your components and export that. Using export default directly causes a warning to appear when hot reloading.

Reply:
Good to know!! Thanks for the awareness 👍